### ISSUE TITLE
Not recognzing running instance of PoE when using Steam

### ISSUE THREAD

**Issue by Tartarus762** [nomis51/Menagerie#31]
I'm using the steam client.

The log is just filled with the following text over and over:

```
2021-07-31 13:12:57.007 +10:00 [INF] Unable to find any known PoE processes
2021-07-31 13:12:57.007 +10:00 [INF] Waiting 5 seconds before retry
2021-07-31 13:12:57.068 +10:00 [INF] Getting clipboard value
2021-07-31 13:12:57.569 +10:00 [INF] Getting clipboard value
2021-07-31 13:12:58.070 +10:00 [INF] Getting clipboard value
2021-07-31 13:12:58.571 +10:00 [INF] Getting clipboard value
2021-07-31 13:12:59.071 +10:00 [INF] Getting clipboard value
2021-07-31 13:12:59.572 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:00.072 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:00.573 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:01.073 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:01.574 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:02.009 +10:00 [INF] Verify client file exists
2021-07-31 13:13:02.011 +10:00 [INF] Verify client file exists
2021-07-31 13:13:02.011 +10:00 [INF] Unable to find any known PoE processes
2021-07-31 13:13:02.011 +10:00 [INF] Waiting 5 seconds before retry
2021-07-31 13:13:02.084 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:02.584 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:03.096 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:03.601 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:04.114 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:04.614 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:05.113 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:05.628 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:06.128 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:06.629 +10:00 [INF] Getting clipboard value
2021-07-31 13:13:07.020 +10:00 [INF] Verify client file exists
2021-07-31 13:13:07.022 +10:00 [INF] Verify client file exists
2021-07-31 13:13:07.022 +10:00 [INF] Unable to find any known PoE processes
```

I started Menagerie after starting poe and have tried restarting it.

The overlay shows up, but incoming and outgoing whispers do not work.

**Comment by nomis51** [comment]
What version of the app are you using ?
What's the name of the Steam client in the installation directory ?

![image](https://user-images.githubusercontent.com/25111613/127742669-6dd1f5f9-36af-409e-a068-e3e28b4d41e0.png)
(Btw that's the Standalone client version)

Currently the app is looking for something called `PathOfExile_x64Steam`. GGG changed the name of the Standalone client couple patches ago when they removed the 32 bits client. That might be the probleme here. Since, I'm not using the Steam client, I didn't noticed.

**Comment by Tartarus762** [comment]
Hi,

It appears that it shows up as PathOfExileSteam in task manager if that helps.

![image](https://user-images.githubusercontent.com/19199618/127759159-3e9e9dd6-4f0f-4104-b636-e08c46a7a3a5.png)

The issue still occurs after updating to 1.4.5

**Comment by nomis51** [comment]
Fixed in  1.4.6
